.github/workflows/jekyll-gh-pages.yml

```
@@ -0,0 +1,51 @@
+name: Deploy Jekyll with GitHub Pages dependencies preinstalled
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["new-design"]
```

Branch to change from "new-design" to "main" right before merging this PR.

```suggestion
    branches: ["main"]
```